code/modules/atmospherics/machinery/pipes/smart.dm

```diff
@@ -11,16 +11,21 @@ GLOBAL_LIST_INIT(atmos_components, typecacheof(list(/obj/machinery/atmospherics)
 	construction_type = /obj/item/pipe/quaternary
 	pipe_state = "manifold4w"
 	connection_num = 0
+	var/list/connections
 
 /obj/machinery/atmospherics/pipe/smart/update_pipe_icon()
 	icon = 'icons/obj/atmospherics/pipes/pipes_bitmask.dmi'
 	var/bitfield = NONE
 	var/bits = 0
+	connections = 0
+	connection_num = NONE
```

Comment:
What? isn't connection_num a number? why are you setting it to a bitflag define? I get that it's also 0, but that's not what it's for

Reply:
Oops. Meant to change connections to NONE.

Reply:
Now connections is NONE and connection_num is 0, though I think bits is doing the same thing as connection_num so I might as well remove connection_num.